File: teams/team-1/java-app/src/main/java/com/example/java/Controller.java
package com.example.java;

import com.example.greeting.GreetingService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class Controller {

        @Autowired
        private GreetingService greetingService;

        @GetMapping("/greet")
        public String greet() {
            return greetingService.greet("Java");
        }
}

File: teams/team-1/kotlin-app/src/main/kotlin/com/example/kotlin/Controller.kt
package com.example.kotlin

import com.example.greeting.GreetingService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class Controller {
    @Autowired
    private val greetingService: GreetingService? = null

    @GetMapping("/greet")
    fun greet(): String {
        return greetingService!!.greet("Kotlin")
    }
}
